enterprise_access/settings/base.py

```diff
@@ -55,6 +55,7 @@ def root(*path_fragments):
     'rest_framework',
     'rest_framework_swagger',
     'rules.apps.AutodiscoverRulesConfig',
+    'simple_history',
```

Comment:
Interestingly, simple_history middleware was installed, but not simple_history itself.